docs/topics/data-analysis/data-analysis-connect-to-db.md

```diff
@@ -1,22 +1,183 @@
 [//]: # (title: Connect and retrieve data from databases)
 
-Simple guide how to start working data sources with Kotlin Notebooks.
+Kotlin Notebook offers capabilities for connecting to and retrieving data from various types of SQL databases, such as 
+MariaDB, PostgreSQL, MySQL, and SQLite. 
+Utilizing the [Kotlin DataFrame library](https://kotlin.github.io/dataframe/gettingstarted.html), Kotlin Notebook can establish 
+connections to databases, execute SQL queries, and import the results for further operations.
+
+For a detailed example, see the [Notebook in the KotlinDataFrame SQL Examples GitHub repository](https://github.com/zaleslaw/KotlinDataFrame-SQL-Examples/blob/master/notebooks/imdb.ipynb).
 
 ## Before you start
 
-* Download and install the latest version of [IntelliJ IDEA](https://www.jetbrains.com/idea/download/index.html).
-* Install the [Kotlin Notebook plugin](https://plugins.jetbrains.com/plugin/16340-kotlin-notebook).
-* Add a dependency to the DataFrame library (?)
+1. Download and install the latest version of [IntelliJ IDEA Ultimate](https://www.jetbrains.com/idea/download/?section=mac).
+2. Install the [Kotlin Notebook plugin](https://plugins.jetbrains.com/plugin/16340-kotlin-notebook) in IntelliJ IDEA.
+
+   > Alternatively, access the Kotlin Notebook plugin from **Settings/Preferences** | **Plugins** | **Marketplace** within IntelliJ IDEA.
+   >
+   {type="tip"}
+
+3. Create a new Kotlin Notebook by selecting **File** | **New** | **Kotlin Notebook**.
+4. In the Kotlin Notebook, import the Kotlin DataFrame library by running the following command:
+
+```kotlin
+%use dataframe
+```
+
+5. Ensure you have access to an SQL database, such as MariaDB or MySQL.
+
+
+## Connect to database
+
+You can connect to and interact with an SQL database using specific functions from the [Kotlin DataFrame library](https://kotlin.github.io/dataframe/gettingstarted.html). 
+You can use `DatabaseConfiguration` to establish a connection to your database and `getSchemaForAllSqlTables()` to retrieve 
+the schema of all tables within it.
+
+Let's look at an example:
+
+1. Open your Kotlin Notebook file (`.ipynb`).
+
+2. Import the Kotlin DataFrame library, which is essential for data manipulation tasks, along with
+the necessary Java libraries for SQL connectivity and utility functions: 
+
+```kotlin
```

Comment:
Hi, we updated our workflow/docs a little bit here https://kotlin.github.io/dataframe/readsqldatabases.html#getting-started-with-notebooks

Very important for user to run cell with the Database dependency earlier, before `use dataframe`

Please update the docs